projects/Amlogic/devices/S905/patches/crazycat/driver.dvb.crazycat-01-add-S905-specific-to-backports.patch

```diff
@@ -0,0 +1,13 @@
+--- a/backports/backports.txt
```

Comment:
shouldn't this be moved to `LE/packages/linux-driver-addons/dvb/crazycat/patches/amlogic` ? doesn't makes sense from pov to spread the patches unnecessarily - including all other files at `projects/Amlogic/devices/S905/patches/crazycat/` and `projects/Amlogic/devices/S905/patches/crazycat/`

Reply:
This is really specific for S905. Wetek dvb driver is built only for S905.